src/components/util/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -36,6 +36,7 @@ interface Props {
 	autofocus?: boolean;
 	onSubmit: (nextValue: string) => void;
 	onRemove?: () => void;
+	focusBubbles?: (event: React.KeyboardEvent<HTMLInputElement>) => void;
```

Comment:
let's rename it to generic name onKeyDown since Autocomplete is used not only for bubbles